releng/release-tool/release-tool.go

```diff
@@ -1271,3 +1277,70 @@ func main() {
 		}
 	}
 }
+
+func updatePhase2Jobs(orgRepo, fullJobConfig string) (string, error) {
+	fileContent, err := ioutil.ReadFile(fullJobConfig)
```

Comment:
What is the reason why you don't use [config.ReadJobConfig](https://github.com/kubernetes/test-infra/blob/c5fbbe779eac4753c6c1a13dda1a89a07de65f32/prow/config/config.go#L1674) from kubernetes/test-infra? I think that could simplify the code, i.e. making unnecessary all the type conversion?

Usage example: https://github.com/kubevirt/project-infra/blob/a940953303f34b402f70d6b45c840fce519f75b7/robots/pkg/kubevirt/cmd/require/presubmits.go#L96

Reply:
i think that when i tried something alike it reordered the fields and did a mess
i can try to make sure with this function

Reply:
Yeah, that's the func I meant. I don't know if it matters much that it reorders the fields, since this is from perspective of the forked jobs, which aren't yet committed anyway, no?

Reply:
Forgot how i tested it on the past, and what was the results tbh

Tried now to trigger only the func from the [diff](https://github.com/kubevirt/project-infra/compare/c42e6a20402114f19a49a528539ff35c1359e6b8..ac8a5d6a7e206ff8c35ce4e03c7b0cf4a75e8980#diff-585313f6146df52ade10307f2f7bd15a57cddd496439c6cc2d8c1cdd30eb7562L1290)

```
	fullJobConfig := fmt.Sprintf("%s/github/ci/prow-deploy/files/jobs/%s/%s/%s-presubmits.yaml",
		"/home/oshoval/go/src/github.com/kubevirt/project-infra", r.org, r.repo, r.repo)
	updateJobConfig, err := updatePhase2Jobs(r.org, r.repo, fullJobConfig)
	if err != nil {
		log.Fatalf("ERROR %s", err)
	}

	log.Printf("updateJobConfig: %s", updateJobConfig)
```

The load fails when it validates the presubmits
```
ERROR [invalid presubmit job pull-kubevirt-e2e-k8s-1.29-sig-performance: invalid decoration config: utility image config is not specified, invalid presubmit job pull-kubevirt-e2e-windows2016: invalid decoration config: utility image config is not specified, invalid presubmit job pull-kubevirt-e2e-kind-1.27-vgpu: invalid decoration config:
```
It calls ... validateDecoration which eventually fails about the decoration

I think the current way the PR suggests is easiest

Reply:
Don't remember how i tested it exactly, 
when i test it now it totally mess the file (did some tricks there to tests it quick and dirty)
IIRC, at the past it was either messing it or added tons of nil fields and this was the reason i dropped it
The current solution is the cleanest i found tbh in matter of the output and also the code is pretty straight forward and has unit tests.